### PR TITLE
doc/kb/tips/skippable_tasks/index

### PR DESCRIPTION
**Pull request by erikfTW**
This PR includes the skippable tasks tip.

Here we have the original tips:

- [Skippable tasks](https://www.taipy.io/tips/the-skippable-tasks/)

### CODE REVIEW

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+![Setting up Data Nodes for Tasks](skippable_tasks_2.png){width=100%}
+
+Before you start using skippable tasks, it's important to configure your tasks correctly 
+with their Data nodes. You can refer to our previous article or 
+documentation for more details on [Data nodes](https://www.taipy.io/tips/the-data-nodes/).
+
+For instance, let's say you have a function like **multiply_and_add()** that takes two parameters 
+and returns two values. How can you represent this function as a Taipy Task?
+
+```python linenums="1"
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: When the code is not explained line by line, or when no line is referred to in the text, we don't need to set the line number.
```suggestion
```python
```

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+It contains:
+
+- Input and output Data nodes.
+- The Python function that you've defined and want to use.
+
+![Setting up Data Nodes for Tasks](skippable_tasks_2.png){width=100%}
+
+Before you start using skippable tasks, it's important to configure your tasks correctly 
+with their Data nodes. You can refer to our previous article or 
+documentation for more details on [Data nodes](https://www.taipy.io/tips/the-data-nodes/).
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Please use a relative path to the documentation version of the tip.

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+In the animation below:
+
+- The first tab corresponds to the creation of the configuration graphically using Taipy Studio.
+- The second tab corresponds to the creation of the very same configuration programmatically.
+
+The order in which you supply Data nodes to the Task is critical. 
+Taipy calls the function using the parameters in the same order as the Data nodes, 
+and the results are returned in that exact order.
+
+<iframe width="640" height="360" src="https://www.taipy.io/wp-content/uploads/2023/04/full_config.mp4?_=1" frameborder="0" allowfullscreen></iframe>
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Can we host this file on taipy doc?

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+
+```python linenums="1"
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: When the code is not explained line by line, we don't need to set the line number.

```suggestion
```python
```

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+task_cfg = Config.configure_task(&amp;amp;amp;quot;multiply_and_add&amp;amp;amp;quot;,
+                                  function=multiply_and_add,
+                                  input=[nb_1_cfg, nb_2_cfg],
+                                  output=[sum_cfg, product_cfg], skippable=True)
+...
+```
+
+With the code below, we create and submit an instance of this scenario configuration.
+
+```python linenums="1"
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: same remark

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+scenario = tp.create_scenario(scenario_cfg)
+tp.submit(scenario)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;, scenario.sum.read())
+```
+
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_4.png){width=100%}
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Please replace picture by console block.

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+
+The line below is resubmitting the scenario, but please note that I haven't made 
+any changes to my input Data nodes in any way.
+
+```python linenums="1"
+tp.submit(scenario)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;,scenario.sum.read())
+```
+
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_5.png){width=100%}
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Please replace picture by console block.

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+```
+
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_4.png){width=100%}
+
+The task associated with my Task has been completed, which means that my function has been executed.
+
+The line below is resubmitting the scenario, but please note that I haven't made 
+any changes to my input Data nodes in any way.
+
+```python linenums="1"
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Same remark

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_5.png){width=100%}
+
+As expected, Taipy is skipping the Task because the input parameters haven't changed. 
+If there are multiple tasks in this scenario, Taipy may skip several of them.
+
+The code below shows what happens when we submit the scenario after making a change to an input Data node. 
+In this case, the value of **nb_1** is updated from 21 to 42.
+
+```python linenums="1"
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Same remark

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+scenario.nb_1.write(42)
+tp.submit(scenario)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;, scenario.sum.read())
+```
+
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_6.png){width=100%}
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Same remark

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+it can also occur when creating and submitting a completely new scenario with Global Data nodes.
+
+For instance, if you want to preprocess a raw data set and make the result accessible globally 
+across the entire application, you can change the scope of both Data nodes (the raw data set and the result) to Global. 
+This means that all scenarios will share these Data nodes, and the Task related to this operation might be 
+skipped across different scenarios.
+
+Let's revisit our previous code and modify the Data nodes to have a Global scope.
+
+<iframe width="640" height="360" src="https://www.taipy.io/wp-content/uploads/2023/03/predict.mp4?_=3" frameborder="0" allowfullscreen></iframe>
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Same remark

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+
+```python linenums="1"
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Same remark

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 13]
```diff
@@ -0,0 +1,208 @@
...
+                              predict,
+                              model_cfg,
+                              predictions_cfg)
+scenario_cfg = Config.configure_scenario_from_tasks(&amp;amp;amp;quot;scenario&amp;amp;amp;quot;, [task_cfg])
+```
+
+The first line creates a scenario consisting of Data nodes, tasks, and pipelines. 
+Following this, we submit it.
+
+```python linenums="1"
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Same remark

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 14]
```diff
@@ -0,0 +1,208 @@
...
+from taipy.config import Config
+model_cfg = Config.configure_data_node(&amp;amp;amp;quot;model&amp;amp;amp;quot;, 
+                                       default_path=&amp;amp;amp;quot;model.p&amp;amp;amp;quot;)
+predictions_cfg = Config.configure_data_node(&amp;amp;amp;quot;predictions&amp;amp;amp;quot;)
+task_cfg = Config.configure_task(&amp;amp;amp;quot;task&amp;amp;amp;quot;,
+                              predict,
+                              model_cfg,
+                              predictions_cfg)
+scenario_cfg = Config.configure_scenario_from_tasks(&amp;amp;amp;quot;scenario&amp;amp;amp;quot;, [task_cfg])
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: The code formatting has been messed up. Please remove special characters.

Plus can you replace `configure_sceanrio_from_task` by `configure_scenario`. This is the new name in the version 3 to be released soon.

```suggestion
from taipy.config import Config
model_cfg = Config.configure_data_node("model", default_path="model.p")
predictions_cfg = Config.configure_data_node("predictions")
task_cfg = Config.configure_task("task", predict, model_cfg, predictions_cfg)
scenario_cfg = Config.configure_scenario("scenario", [task_cfg])
```

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 15]
```diff
@@ -0,0 +1,208 @@
...
+This feature can significantly improve the performance of your pipeline 
+by preventing unnecessary computations, which saves time and resources.
+
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_3.png){width=70%}
+
+## Use Case
+
+Let’s take the previous execution graph and set **skippable=True** to our Task.
+
+<iframe width="640" height="360" src="https://www.taipy.io/wp-content/uploads/2023/04/skippable.mp4?_=2" frameborder="0" allowfullscreen></iframe>
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: same remark

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 16]
```diff
@@ -0,0 +1,208 @@
...
+                                  output=[sum_cfg, product_cfg], skippable=True)
+...
+```
+
+With the code below, we create and submit an instance of this scenario configuration.
+
+```python linenums="1"
+scenario = tp.create_scenario(scenario_cfg)
+tp.submit(scenario)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;, scenario.sum.read())
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Please clean the string code.

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 17]
```diff
@@ -0,0 +1,208 @@
...
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_4.png){width=100%}
+
+The task associated with my Task has been completed, which means that my function has been executed.
+
+The line below is resubmitting the scenario, but please note that I haven't made 
+any changes to my input Data nodes in any way.
+
+```python linenums="1"
+tp.submit(scenario)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;,scenario.sum.read())
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md:119]: Please clean the string code.

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 18]
```diff
@@ -0,0 +1,208 @@
...
+As expected, Taipy is skipping the Task because the input parameters haven't changed. 
+If there are multiple tasks in this scenario, Taipy may skip several of them.
+
+The code below shows what happens when we submit the scenario after making a change to an input Data node. 
+In this case, the value of **nb_1** is updated from 21 to 42.
+
+```python linenums="1"
+scenario.nb_1.write(42)
+tp.submit(scenario)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;, scenario.sum.read())
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Please clean the string code.

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 19]
```diff
@@ -0,0 +1,208 @@
...
+```python linenums="1"
+from taipy.config import Config
+model_cfg = Config.configure_data_node(&amp;amp;amp;quot;model&amp;amp;amp;quot;, 
+                                       default_path=&amp;amp;amp;quot;model.p&amp;amp;amp;quot;)
+predictions_cfg = Config.configure_data_node(&amp;amp;amp;quot;predictions&amp;amp;amp;quot;)
+task_cfg = Config.configure_task(&amp;amp;amp;quot;task&amp;amp;amp;quot;,
+                              predict,
+                              model_cfg,
+                              predictions_cfg)
+scenario_cfg = Config.configure_scenario_from_tasks(&amp;amp;amp;quot;scenario&amp;amp;amp;quot;, [task_cfg])
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Same remark on the string code that is messed up + same remark on configure_scenario_from_tasks

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 20]
```diff
@@ -0,0 +1,208 @@
...
+```
+
+The first line creates a scenario consisting of Data nodes, tasks, and pipelines. 
+Following this, we submit it.
+
+```python linenums="1"
+scenario_1 = tp.create_scenario(scenario_cfg)
+tp.submit(scenario_1)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;, scenario_1.sum.read())
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Please clean the string code.

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 21]
```diff
@@ -0,0 +1,208 @@
...
+```
+
+![Using Global Data Nodes](skippable_tasks_7.png){width=100%}
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Please replace the picture by console code block

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 22]
```diff
@@ -0,0 +1,208 @@
...
+
+![Using Global Data Nodes](skippable_tasks_8.png){width=100%}
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Please replace the picture by console code block

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 23]
```diff
@@ -0,0 +1,208 @@
...
+The only task has been completed, and the results have been computed.
+
+Now, let's create another scenario. 
+This new scenario won't create new Global Data nodes (both input and output). 
+Instead, it will reuse the ones that were created by **scenario_1**.
+
+```python linenums="1"
+scenario_2 = tp.create_scenario(scenario_cfg)
+tp.submit(scenario_2)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;, scenario_2.sum.read())
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Please clean the string code.

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 24]
```diff
@@ -0,0 +1,208 @@
...
+
+![Using Global Data Nodes](skippable_tasks_8.png){width=100%}
+
+The only task has been completed, and the results have been computed.
+
+Now, let's create another scenario. 
+This new scenario won't create new Global Data nodes (both input and output). 
+Instead, it will reuse the ones that were created by **scenario_1**.
+
+```python linenums="1"
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Same remark

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 25]
```diff
@@ -0,0 +1,208 @@
...
+scenario_2 = tp.create_scenario(scenario_cfg)
+tp.submit(scenario_2)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;, scenario_2.sum.read())
+```
+
+![Using Global Data Nodes](skippable_tasks_9.png){width=100%}
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Please replace the picture by console code block

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 26]
```diff
@@ -0,0 +1,208 @@
...
+
+Taipy reuses the existing Data nodes and skips the Task if the input Data nodes have not changed.
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]:
```suggestion
Taipy skips the Task if the input Data nodes have not changed and reuses the existing output's Data.
```

**Review by jrobinAV — CHANGES_REQUESTED** [review]

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 27]
```diff
@@ -0,0 +1,208 @@
...
+with interactive interfaces that handle data. 
+This article is all about Taipy's back-end capabilities. 
+Just as a reminder, the main goal of Taipy's back-end is to help you build 
+and manage complex workflows, such as data processing pipelines.
+
+![Skippable Tasks](skippable_tasks.png){width=100%}
+
+When engineers create Directed Acyclic Graphs (DAGs), they often leave out certain tasks. 
+This common practice allows tasks to be orchestrated in a more dynamic and advanced way. 
+One of the key features of Taipy's back-end is skippable tasks, which can be skipped under specific conditions.
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: skippable -> \*skippable\*

File: docs/knowledge_base/tips/skippable_tasks/index.md (hunk 3 above)
Comment (FabienLelaquais) [reply docs/knowledge_base/tips/skippable_tasks/index.md]: I hope so!!

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 28]
```diff
@@ -0,0 +1,208 @@
...
+                                       default_path=&amp;amp;amp;quot;model.p&amp;amp;amp;quot;)
+predictions_cfg = Config.configure_data_node(&amp;amp;amp;quot;predictions&amp;amp;amp;quot;)
+task_cfg = Config.configure_task(&amp;amp;amp;quot;task&amp;amp;amp;quot;,
+                              predict,
+                              model_cfg,
+                              predictions_cfg)
+scenario_cfg = Config.configure_scenario_from_tasks(&amp;amp;amp;quot;scenario&amp;amp;amp;quot;, [task_cfg])
+```
+
+In this example, the **multiply_and_add()** function takes two parameters (**nb1** and **nb2**).
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Functions and variables use \* everywhere and not \*\*. 
This also happens somewhere above this files.
Please homogenize.

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 29]
```diff
@@ -0,0 +1,208 @@
...
+In this example, the **multiply_and_add()** function takes two parameters (**nb1** and **nb2**). 
+It returns two values: the product and the sum. 
+We create configurations for the input Data nodes (**nb_1_cfg** and **nb_2_cfg** in this order) 
+and the output Data nodes (**sum_cfg** and **product_cfg**).
+
+Finally, we configure the Task with the appropriate input and output Data nodes.
+
+## Leveraging Skippability in Taipy Tasks
+
+Skippability is an optional setting that you can enable when configuring a Task.
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: Break line. Add \<br/\>

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 30]
```diff
@@ -0,0 +1,208 @@
...
+In other words, if running the task again would produce the same output, it is skipped.
+
+This feature can significantly improve the performance of your pipeline 
+by preventing unnecessary computations, which saves time and resources.
+
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_3.png){width=70%}
+
+## Use Case
+
+Let’s take the previous execution graph and set **skippable=True** to our Task.
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: "\*\*skippable=True\*\*" -> "\`skippable=True\`"

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 31]
```diff
@@ -0,0 +1,208 @@
...
+
+Taipy reuses the existing Data nodes and skips the Task if the input Data nodes have not changed.
+
+## Manual Changes
+
+Taipy uses the last modification date of a Data Node to determine if an input Data node has been changed. 
+This modification date can be altered in three ways:
+
+- When you submit a Task or a scenario, it automatically updates the last modification date.
+- You can manually update it by writing: `<Data node>.write(…)`.
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tips/skippable_tasks/index.md]: A link to `DataNode.write()` would I guess be nice...

**Review by FabienLelaquais — APPROVED** [review]
Typography conventions

File: docs/knowledge_base/tips/skippable_tasks/index.md (hunk 3 above)
Comment (erikfTW) [reply docs/knowledge_base/tips/skippable_tasks/index.md]: Let me try it! :)

File: docs/knowledge_base/tips/skippable_tasks/index.md (hunk 14 above)
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]:
```suggestion
model_cfg = Config.configure_data_node("model", default_path="model.p")
predictions_cfg = Config.configure_data_node("predictions")
task_cfg = Config.configure_task("task", predict, model_cfg, predictions_cfg)
scenario_cfg = Config.configure_scenario_from_tasks("scenario", [task_cfg])
```

File: docs/knowledge_base/tips/skippable_tasks/index.md [hunk 32]
```diff
@@ -0,0 +1,208 @@
...
+Let’s take the previous execution graph and set **skippable=True** to our Task.
+
+<iframe width="640" height="360" src="https://www.taipy.io/wp-content/uploads/2023/04/skippable.mp4?_=2" frameborder="0" allowfullscreen></iframe>
+
+```python linenums="1"
+...
+task_cfg = Config.configure_task(&amp;amp;amp;quot;multiply_and_add&amp;amp;amp;quot;,
+                                  function=multiply_and_add,
+                                  input=[nb_1_cfg, nb_2_cfg],
+                                  output=[sum_cfg, product_cfg], skippable=True)
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]:
```suggestion
task_cfg = Config.configure_task("multiply_and_add",
                                  function=multiply_and_add,
                                  input=[nb_1_cfg, nb_2_cfg],
                                  output=[sum_cfg, product_cfg], skippable=True)
```

File: docs/knowledge_base/tips/skippable_tasks/index.md (hunk 16 above)
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]:
```suggestion
print("Results (sum):", scenario.sum.read())
```

File: docs/knowledge_base/tips/skippable_tasks/index.md (hunk 17 above)
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md:119]:
```suggestion
print("Results (sum):", scenario.sum.read())
```

File: docs/knowledge_base/tips/skippable_tasks/index.md (hunk 18 above)
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]:
```suggestion
print("Results (sum):", scenario.sum.read())
```

File: docs/knowledge_base/tips/skippable_tasks/index.md (hunk 19 above)
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]:
```suggestion
model_cfg = Config.configure_data_node("model",  default_path="model.p")
predictions_cfg = Config.configure_data_node("predictions")
task_cfg = Config.configure_task("task", predict, model_cfg, predictions_cfg)
scenario_cfg = Config.configure_scenario_from_tasks("scenario", [task_cfg])
```

File: docs/knowledge_base/tips/skippable_tasks/index.md (hunk 20 above)
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]:
```suggestion
print("Results (sum):", scenario_1.sum.read())
```

File: docs/knowledge_base/tips/skippable_tasks/index.md (hunk 23 above)
Comment (jrobinAV) [inline docs/knowledge_base/tips/skippable_tasks/index.md]:
```suggestion
print("Results (sum):", scenario_2.sum.read())
```

File: docs/knowledge_base/tips/skippable_tasks/index.md (hunk 3 above)
Comment (erikfTW) [reply docs/knowledge_base/tips/skippable_tasks/index.md]: It works!

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**